init.sh

```diff
@@ -32,7 +32,34 @@ if [ ! -d "$install_directory" ]; then
     mkdir -p "$install_directory"
 fi
 
-# Copy the contents of the starter pack repository to the installation directory
+# Check if .gitignore exists in the destination directory
+# If exists, append the contents of the source .gitignore to the destination .gitignore
+if [ -f "$install_directory/.gitignore" ]; then
+    echo "ACTION REQUIRED: .gitignore already exists in the destination directory. Check the contents before saving the file!"
```

Comment:
```suggestion
    echo "ACTION REQUIRED: .gitignore already exists in the destination directory."
```
The file will be saved automatically, so I think that statement is a bit misleading. I'd leave it out here and instead add something to the message if users choose to skip. (If the user chooses to append, I wouldn't expect any issues.)